components/list.js

```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import SelectBox from "./slectBox";
```

Comment:
looks like you had a spelling error here

Reply:
I don't see it anymore, I guess you fixed it